File: 2058-ConcatenationOfArray/2058-ConcatenationOfArray.py
# Last updated: 3/25/2026, 9:03:11 AM
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        n = len(nums)
        ans = [0] * (2 * n)
        for i in range(n):
            ans[i] = ans[i + n] = nums[i]
        
        return ans

File: 2058-ConcatenationOfArray/2058-ConcatenationOfArray.java
// Last updated: 3/25/2026, 9:03:16 AM
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n = nums.length;
        int[] ans = new int[2*n];
        for(int i = 0 ; i < n ; i++) {
            int j = i;
            ans[i] = nums[i];
            ans[j+n] = nums[i];
        }
        

        return ans;
    }
}